### ISSUE TITLE
[9.0] Travis failing on SASS execution

### ISSUE THREAD

**Issue by lasley** [OCA/website#282]
First reported in #279, also noticed in #277. I rebuilt 9.0 branch on Travis, and it is now failing in the same way: https://travis-ci.org/OCA/website/builds/167816895

```
2016-11-15 17:32:13,517 5314 ERROR openerp_test openerp.addons.base.ir.ir_qweb: Could not execute command 'sass'
```

**Comment by pedrobaeza** [comment]
We have faced the same issue and tried to fix it with https://github.com/OCA/website/pull/277/files#diff-354f30a63fb0907d4ad57269548329e3R32

**Comment by lasley** [comment]
You're still getting SASS issues there too though (albeit not as bad):

```
2016-11-15 11:32:27,007 7442 WARNING openerp_test openerp.addons.base.ir.ir_qweb: LoadError: cannot load such file -- bootstrap-sass
```

Do we have any idea what the SASS is actually being used for? I'm confused why bootstrap-sass is even a thing; Odoo uses bootstrap-less 😕

**Comment by pedrobaeza** [comment]
AFAIK they used both. Then we also need to install another thing more...

**Comment by lasley** [comment]
Closing, fixed in #284
